nodecg-io-android/package.json

```
@@ -0,0 +1,42 @@
+{
+    "name": "nodecg-io-android",
+    "version": "0.1.0",
+    "description": "Allows to connect to an android device via adb.",
+    "homepage": "https://nodecg.io/samples/android",
+    "author": {
+        "name": "noeppi_noeppi",
+        "url": "https://github.com/noeppi-noeppi"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/codeoverflow-org/nodecg-io.git",
+        "directory": "nodecg-io-android"
+    },
+    "scripts": {
+        "build": "tsc -b",
+        "watch": "tsc -b -w",
+        "clean": "tsc -b --clean"
+    },
+    "main": "extension",
+    "keywords": [
+        "",
+        "nodecg-bundle"
+    ],
+    "nodecg": {
+        "compatibleRange": "^1.1.1",
+        "bundleDependencies": {
+            "nodecg-io-core": "0.1.0"
+        }
+    },
+    "license": "MIT",
+    "devDependencies": {
+        "@types/node": "^14.14.13",
+        "nodecg": "^1.7.4",
+        "typescript": "^4.1.3"
+    },
+    "dependencies": {
+        "nodecg-io-core": "0.1.0",
```

Since you've started with the android service there were some changes to the template package.json and the nodecg-io label and the semver major operators are missing here. So add them to make this consistent across all services.
```suggestion
        "nodecg-io",
        "nodecg-bundle"
    ],
    "nodecg": {
        "compatibleRange": "^1.1.1",
        "bundleDependencies": {
            "nodecg-io-core": "^0.1.0"
        }
    },
    "license": "MIT",
    "devDependencies": {
        "@types/node": "^14.14.13",
        "nodecg": "^1.7.4",
        "typescript": "^4.1.3"
    },
    "dependencies": {
        "nodecg-io-core": "^0.1.0",
```